src/core_modules/capture-core/components/WorkingLists/TeiWorkingLists/epics/teiViewEpics/helpers/getTeiListData/convertToClientTeis.js

```diff
@@ -29,7 +30,9 @@ export const convertToClientTeis = (
                     return {
                         id,
                         value: convertServerToClient(value, type),
-                        urlPath: `/trackedEntityInstances/${tei.trackedEntity}/${id}/image`,
+                        urlPath: featureAvailable(FEATURES.trackerImageEndpoint) ?
+                            `/tracker/trackedEntities/${tei.trackedEntity}/attributes/${id}/image?program=${programId}&dimension=small` :
+                            `/trackedEntityInstances/${tei.trackedEntity}/${id}/image`,
```

Comment:
Can we only add the urlPath if it is an image? Would improve readability quite a bit.

Also, in https://github.com/dhis2/capture-app/blob/c4b29c4b2c15195d1193636e8ca9d72536b53b9f/src/core_modules/capture-core/components/WorkingLists/TeiWorkingLists/epics/teiViewEpics/helpers/getListDataCommon/getSubvalues.js#L55 **and others**, can we have a `previewUrl` and `fullSizedUrl` instead of the single `url` and remove the replace statement in `PreviewImage.component`. Would be more robust that way and the "UI-library" will not be responsible for computing URLs.